src/client/theme-default/styles/base.css

```diff
@@ -250,3 +250,9 @@ mjx-container {
 mjx-container > svg {
   margin: auto;
 }
+
+@media print {
+  .screen-only {
```

Comment:
maybe move this to src/client/theme-default/styles/utils.css ?

Reply:
done